### PR TITLE
Fixed datetime method

### PR DESCRIPTION
**Pull request by DonutPirate**

### CODE REVIEW

File: goco-transit/src/services/date-service.js [hunk 1]
```diff
@@ -5,16 +5,17 @@
  * @param {datetime} datetime to convert
  */
 const getDate = (datetime) => {
-  let date = new Date(datetime.split("T")[0]);
-  return date.getMonth() + "/" + date.getDate() + "/" + date.getFullYear().toString().substr(-2);
+  console.log(datetime)
```
Comment (Natoman95) [inline goco-transit/src/services/date-service.js]: want to remove this log dingus? I've noticed some other garbage showing up in the console as well. Remove these when you're done debugging N0ob

Comment (DonutPirate) [reply goco-transit/src/services/date-service.js]: no u

Comment (Natoman95) [reply goco-transit/src/services/date-service.js]: I won't merge until you fix it

Comment (DonutPirate) [reply goco-transit/src/services/date-service.js]: fixed

**Pull request merged**